### DISCUSSION TITLE
vs_runtime

### DISCUSSION THREAD

**Discussion by lichanglin688** [xmake-io/xmake#3957] [Q&A]
### Is your feature request related to a problem? Please describe.

vs_runtime通常不是MD吗

### Describe the solution you'd like

vs_runtime为什么默认值是MT

**Comment by waruqi** [comment]
MD 那个是 vs vcproj 里面设置的默认值，但是 cl.exe 你如果不加任何 flags 编译，默认就是 MT

而 xmake 默认是不会自动加额外的 flags 的，所以默认也就是 MT （没显式加 /MT），尽可能最小化内置 flags 。。所有 flags 由用户自己控制，外面自己设置。。

[a comment by an automated account is not shown]
